akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/users/ChangeUserAwayReqMsgHdlr.scala

```diff
@@ -44,6 +46,9 @@ trait ChangeUserAwayReqMsgHdlr extends RightsManagementTrait {
         outGW.send(MsgBuilder.buildUserEmojiChangedEvtMsg(liveMeeting.props.meetingProp.intId, msg.body.userId, "none"))
       }
 
+      val chatMsg = s"${user.name} is " + (if (msg.body.away) "now " else "no longer ") + "away"
+      ChatMessageDAO.insertSystemMsg(liveMeeting.props.meetingProp.intId, GroupChatApp.MAIN_PUBLIC_CHAT, chatMsg, GroupChatMessageType.SYSTEM, Map(), "")
```

Comment:
It's not a good idea to write the message here. Because akka-apps doesn't support internationalization.
Here you can just send an empty message and add the message information in the `Map()`, just like `GroupChatMessageType.POLL` for example.

This way the client will have the message type and the necessary information. And it will be able to write the message using `intl.formatMessage`.